src/index.js

```diff
@@ -0,0 +1,80 @@
+// import _ from 'lodash';
+import './style.css';
+import todoTasks from './modules/task';
+import {
+  addTask,
+  deleteTask,
+  addNewTask,
+  toggleEditMode,
+  saveToLocalStorage,
+} from './modules/addremove';
+
+const listDiv = document.querySelector('.list-items');
+const clearBtn = document.querySelector('.clear');
+const inputField = document.querySelector('.task');
+
+const buildTask = () => {
+  listDiv.innerHTML = '';
+  todoTasks.forEach((task) => {
+    addTask(task.description, task.index, task.completed);
+  });
+
+  const checkboxes = document.querySelectorAll('.check-box');
+  checkboxes.forEach((checkbox) => {
+    checkbox.addEventListener('change', () => {
+      const index = Number(checkbox.id.split('-')[0]);
+      const status = todoTasks[index - 1].completed;
+      todoTasks[index - 1].completed = !status;
+      saveToLocalStorage();
+    });
+  });
+  saveToLocalStorage();
+};
```

Comment:
As per the project requirements, methods related to the status updates `(completed: true/false)` should be in a new javascript file that should be imported as a module. Kindly move the code related to this functionality to a new file and then import that as a module. ⬇️
![image](https://github.com/albertkantwi/to_do_list/assets/33268927/0ba81c0d-1fee-4bd4-a2d9-86ed9d881512)